crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -5162,6 +5164,16 @@ impl<'a> Checker<'a> {
                     Location::new(expr.location.row(), expr.location.column()),
                 ));
 
+                if pydocstyle::helpers::should_ignore_docstring(contents) {
+                    warn_user!(
+                        "Docstring at {}:{}:{} contains implicit string concatenation; ignoring...",
+                        relativize_path(self.path),
+                        expr.location.row(),
+                        expr.location.column() + 1
+                    );
+                    continue;
```

Comment:
Ideally this would be a diagnostic but I'm not sure that it fits any of our existing diagnostics under the `pydocstyle` category.